### PR TITLE
Add shortcut method dd-mm-yyyy for American date format, similar to t…

### PR DESCRIPTION
**Pull request by Xliff**
…he yyyy-dd-mm method.

For those who really want to make their string dates harder to deal with (and I'm American and biased!)

### CODE REVIEW

File: src/core/Dateish.pm [hunk 1]
```diff
@@ -107,6 +107,7 @@ my role Dateish {
     }
 
     method yyyy-mm-dd() { sprintf '%04d-%02d-%02d',$!year,$!month,$!day }
+    method mm-dd-yyyy() { sprintf '%02d-%02d-%02d',$!month,$!day,$!year }
```
Comment (jonathanstowe) [inline src/core/Dateish.pm]: should that last one be %04d ?

**Comment by AlexDaniel** [comment]
So what's the reason to add this particular format?

**Comment by labster** [comment]
This seems like extra clutter for the core.  I'm American and I never use this date format, except when filling out weird paper forms by hand.  I don't understand why this wouldn't be in a module, or even an augment block.

**Comment by zoffixznet** [comment]
:-1: From me on this one. It's not a standard way to write a day and as others pointed out, just adds clutter.

**Comment by Xliff** [comment]
Fair enough. Close it.

**Comment by zoffixznet** [comment]
Thanks anyway 😸

**Pull request closed without merging**